### PR TITLE
desktop version with new blocks

### PR DESCRIPTION
**Pull request by AnnaShpak01**
landing new blocks (desktop version)

### CODE REVIEW

**Comment by AnnaShpak01** [comment]
https://trello.com/c/3CGvehx8/38-landingpage-new-blocks

File: gui/landing/_config.yml [hunk 1]
```diff
@@ -139,24 +143,47 @@ social_links:
...
-  - { name: 'Nick Sawinyh', position: 'Advisor', description: 'Blockchain expert since 2017, Co-founder of Defiprime, Key Opinion Leader (KOL), ex-CEO of DexGuru', img: 'ns'}
+  - { name: 'Nick Sawinyh', position: 'Advisor', description: 'Blockchain expert since 2017, Co-founder of <a href="https://defiprime.com/" class="violet">Defiprime</a>, Key Opinion Leader (KOL), ex-CEO of DexGuru', img: 'ns'}
 
 partners2: 
   - {key: chainlink, url: https://chain.link/}
   - {key: telcoin, url: https://www.telco.in/ }
   - {key: uncx, url: https://uncx.network/}
   - {key: paraswap, url: https://www.paraswap.io/}
-#  - {key: 3baseio, url: https://3base.org/ }
+  - {key: 3baseio, url: https://3base.org/ }
```
Comment (Kharabet) [inline gui/landing/_config.yml:186]:
```suggestion
```

the were removed from our partners

File: gui/landing/_includes/achievements.html [hunk 2]
```diff
@@ -46,11 +45,11 @@ <h2 class="section__title">
         <li class="achievements__item">
           <div class="achievements-entry">
             <strong class="achievements-entry__title"
-              >Flow Orchestrator deployed onto</strong
+              >Olas is deployed across</strong
```
Comment (Kharabet) [inline gui/landing/_includes/achievements.html:48]: Olas?

File: gui/landing/_site/404.html [hunk 3]
```diff
@@ -201,6 +203,24 @@
...
+            </a>
+          </li><li class="page-header-links__item">
+            <a
+              href="https://discord.gg/dPW8fzwzz9"
+              class="page-header-links__link page-header-link"
+            ><span class="page-header-link__content">
+                <strong class="page-header-link__title">
+                  Twitch
+                </strong></span>
+            </a>
```
Comment (Kharabet) [inline gui/landing/_site/404.html]: links

File: gui/landing/_site/distribution.html [hunk 4]
```diff
@@ -10,18 +10,20 @@
...
+{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"guru_network"},"description":"Base Layer 3 allows dApps and retail users to natively embed orchestrated AI Agents into their routines and earn Network participant rewards.","headline":"GURU - The AI Orchestration Network","url":"http://localhost:4000/distribution.html"}</script>
 <!-- End Jekyll SEO tag -->
-<link rel="stylesheet" href="/assets/css/style.css?v=20240614123957"><link type="application/atom+xml" rel="alternate" href="/feed.xml" title="GURU Network – The AI Orchestration Network" />
+<link rel="stylesheet" href="/assets/css/style.css?v=20240710125424"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="GURU Network – The AI Orchestration Network" />
 <meta property="og:url" content="https://gurunetwork.ai/" />
   <meta
     property="og:image"
-    content="https://gurunetwork.ai/assets/img/ogp.png?v=20240614123957"
+    content="https://gurunetwork.ai/assets/img/ogp.png?v=20240710125424"
```
Comment (Kharabet) [inline gui/landing/_site/distribution.html]: localhost :eyes:

File: gui/landing/_site/feed.xml [hunk 5]
```diff
@@ -1 +1 @@
-<?xml version="1.0" encoding="utf-8"?><feed xmlns="http://www.w3.org/2005/Atom" ><generator uri="https://jekyllrb.com/" version="4.3.3">Jekyll</generator><link href="/feed.xml" rel="self" type="application/atom+xml" /><link href="/" rel="alternate" type="text/html" /><updated>2024-06-14T12:39:57+02:00</updated><id>/feed.xml</id><title type="html">GURU Network – The AI Orchestration Network</title><author><name>guru_network</name><email>evgeny@dex.guru</email></author></feed>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?><feed xmlns="http://www.w3.org/2005/Atom" ><generator uri="https://jekyllrb.com/" version="4.3.3">Jekyll</generator><link href="http://localhost:4000/feed.xml" rel="self" type="application/atom+xml" /><link href="http://localhost:4000/" rel="alternate" type="text/html" /><updated>2024-07-10T12:54:24+02:00</updated><id>http://localhost:4000/feed.xml</id><title type="html">GURU Network – The AI Orchestration Network</title><author><name>guru_network</name><email>evgeny@dex.guru</email></author></feed>
```
Comment (Kharabet) [inline gui/landing/_site/feed.xml]: localhost

**Pull request merged**